app/presenters/hyrax/image_presenter.rb

```diff
@@ -0,0 +1,8 @@
+# Generated via
+#  `rails generate hyrax:work Image`
+module Hyrax
+  class ImagePresenter < Hyrax::WorkShowPresenter
+    # We do not use this generated ImagePresenter. Instead we use the
```

Comment:
Catch up with Hyrax's work generator, and drop a comment to make it clear this class is unused. I could be convinced to just remove this file, too.